### ISSUE TITLE
[Vue] Scaffold component is missing on Safari

### ISSUE THREAD

**Issue by datdoan** [Sitecore/jss#953]
## Description
New scaffolded component is missing on Safari browser.

## Expected behavior
The  scaffolded component should mounted on page

## Steps To Reproduce

1. Start with the Vue sample app
2. Scaffold a component using `jss scaffold MyComponent`
3. Add the component in `data/routes/en.yml`
4. `jss start` local and open in Safari browser
5. See error message on page "JSS component is missing Vue implementation. See the developer console for more information."

## Your Environment

* Sitecore Version: -
* JSS Version: 19.0.2
* Browser Name and version: 14.1.2 (16611.3.10.1.6)
* Operating System and version (desktop or mobile): MacOS 11.6.4 (20G417)
* Link to your project (if available): -

## Screenshots

![image](https://user-images.githubusercontent.com/3129504/158688119-5893ca3b-b55d-4c54-88bb-d65f0c08e1b0.png)
![image](https://user-images.githubusercontent.com/3129504/158688198-db43cf38-d7fc-4acd-98c1-9e58195882e2.png)

**Comment by markgibbons25** [comment]
You need to run `jss deploy` after you scaffold a new component

**Comment by datdoan** [comment]
> You need to run `jss deploy` after you scaffold a new component

Thanks for the solution, that works.

Didn't have to `css deploy` for it to work/hot reload on Chrome, Brave and Firefox. Could this be a bug or just a limitation of Safari?

**Comment by markgibbons25** [comment]
Yes it does seem like a bug, I didn't realise you were working in disconnected mode, that error would be normal in connected mode if you hadn't done a deploy, but in disconnected mode it's unexpected.

**Comment by datdoan** [comment]
I've recreated the same step with React template, no issue with Safari.
